data/tutorials/en/introOCaml.md

```diff
@@ -0,0 +1,16 @@
+# Introduction to OCaml
+
+OCaml is a versatile and efficient functional programming language that's increasing in popularity across the functional programming community. It just celebrated its 25th anniversary in 2021, as Xavier Leroy released OCaml on May 9, 1996 in collaboration with Jérôme Vouillon, Damien Doligez, and Didier Rémy, and it has steadily grown over the past few decades. Thus far, it has been mostly taught in computer science programs at universities, but we'd like to make it more accessible to the greater programming community. The following documentation gives an introduction to OCaml, teaches the basics, explains the similarities and differences to the more popular imperative languages, and outlines OCaml's libraries and syntax. You'll soon find OCaml is a straightforward, simple language to learn. 
+
+Highly expressive and practical, OCaml is also a compiling language, but it differs from most compilers through the OCaml *toplevel*, a Read-Eval-Print Loop (REPL), allowing iteration. This means you can compile OCaml as you write the code rather than creating an executable afterwards, which assists in finding bugs and mistakes along the way.
+
+OCaml is an important language to learn because it's not only reliable and secure, it's also an excellent choice when building complex software systems. OCaml is compiled from decades of language features like incremental garbage collection, first-class functions, static type-checking, parametric polymorphism, pattern matching, and more. The subsequent documents will expound on these throughout as we progress through the basics and move into more advanced concepts.
```

Comment:
> OCaml is compiled from decades of language

The wording with "compiled" can be slightly confusing here, since "compiled" is used to refer as the compilation of a program in the document.